perfkitbenchmarker/providers/alicloud/util.py

```diff
@@ -72,13 +72,12 @@ def AddTags(resource_id, resource_type, region, **kwargs):
   tag_cmd = ALI_PREFIX + [
       'ecs',
       'AddTags',
-      '--RegionId %s' % region,
-      '--ResourceId %s' % resource_id,
-      '--ResourceType %s' % resource_type]
-  for index, (key, value) in enumerate(six.iteritems(kwargs)):
-    tag_cmd.append('--Tag{0}Key {1} --Tag{0}Value {2}'
-                   .format(index + 1, key, value))
-  tag_cmd = GetEncodedCmd(tag_cmd)
```

Comment:
Why does this no longer need encoding?

Reply:
The `GetEncodedCmd` method was converting a list of commands to a space-separated string, then using shlex to convert it to a shell-like syntax (flag names and arguments separate). That was necessary when the flag names and values were provided as a single list item (using `%s` for concatenation).
The current version already provides the arguments in shell-like syntax (flag names and arguments as separate list items), so encoding is no longer necessary.

Reply:
ok